main.py

```diff
@@ -0,0 +1,515 @@
+"""player for game"""
```

Comment:
Don't forget to add a header/docstring with a short description of what the code contains/does.